ci/jenkins/Jenkinsfile-libres

```diff
@@ -10,30 +10,35 @@ pipeline {
                 sh 'sh ci/jenkins/test_libres_jenkins.sh setup'
             }
         }
-	stage('build libres') {
+        stage('build libres') {
             steps {
                 sh 'sh ci/jenkins/test_libres_jenkins.sh build_libecl'
                 sh 'sh ci/jenkins/test_libres_jenkins.sh build_libres'
             }
         }
-	stage('build res') {
+        stage('build res') {
             steps {
                 sh 'sh ci/jenkins/test_libres_jenkins.sh build_res'
             }
         }
-	stage('run ctest') {
-            steps {
-                sh 'sh ci/jenkins/test_libres_jenkins.sh run_ctest'
-            }
-        }
-	stage('run pytest_equinor') {
-            steps {
-                sh 'sh ci/jenkins/test_libres_jenkins.sh run_pytest_equinor'
-            }
-        }
-	stage('run pytest_normal') {
-            steps {
-                sh 'sh ci/jenkins/test_libres_jenkins.sh run_pytest_normal'
+        stage('tests') {
+            failFast true
```

Comment:
Just wondering what the rationale for failfast here is?

Reply:
If Ctests failed, then there is no reason to continue with the other two.

Reply:
There's a lot of code in ERT that is not using any of the C bindings. So one reason to continue is that the result of those tests might be interesting.

Reply:
okay I will remove failfast